src/vapor.ts

```diff
@@ -0,0 +1,78 @@
+const vaporCommands: Fig.Suggestion[] = [
```

Comment:
```suggestion
const vaporCommands = [
```
This should do the tweak